src/raw.rs

```diff
@@ -445,30 +456,56 @@ pub fn replicate_verbatim(ctx: *mut RedisModuleCtx) -> Status {
     unsafe { RedisModule_ReplicateVerbatim.unwrap()(ctx).into() }
 }
 
+fn load<T>(
```

Comment:
I was wondering if `load` and `load_with_param` could be unified by making it generic over the function argument (which would be optional). Not sure it can, but may be worth trying.

Reply:
👍🏼 Did some improvement - unified using a closure